src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,46 @@
+package tests;
+
+import client.Client;
+import org.junit.jupiter.api.BeforeEach;
+import user.User;
+import utils.authenticationClient.AuthenticationClient;
+import utils.property.PropertyUtil;
+
+import java.util.Properties;
+
+public class BaseTest {
+    public static String readerToken;
+    public static String writerToken;
+    public static User user;
+    public static String incorrectZipCode;
+    private static Client writerClient;
+    private static Client readerClient;
+    private  final Properties clientData = PropertyUtil.getProperties("clientData.properties");
+    private  final Properties httpHost = PropertyUtil.getProperties("httpHost.properties");
+    private  final Properties testData = PropertyUtil.getProperties("testData.properties");
+    @BeforeEach
+    public void init() {
+        writerClient = new Client();
+        writerClient.setUserName(clientData.getProperty("writer.user.name"));
+        writerClient.setUserPassword(clientData.getProperty("writer.user.password"));
+        writerClient.setGrantType(clientData.getProperty("writer.user.grant.type"));
+        writerClient.setUserScope(clientData.getProperty("writer.user.scope"));
```

Comment:
Please create constructor with parameters and create object via constructor

Reply:
fixed